### ISSUE TITLE
[Bug]: Opening game to LAN crashes the game, or changing focus from the chat window

### ISSUE THREAD

**Issue by Raidobw2** [CreeperHost/CreeperHostGui#625]
### Describe the bug

If you lose focus of the Minecraft chat and then go back to it, the game will crash

### Steps to reproduce

This is what I did:
1. Press T to open chat
2. Click Commands
3. Click "Dont ask me again"
4. Type "hello server", and open chat again with T
5. Click on a server console nearby, changing window focus
6. Type "hello client" in the server console
7. Click on the client chat to observe the exchange of words
8. Crash

### Expected behaviour

To not crash

### Screenshots

Sorry, I have none, the window closes

Here is the crash though:
```
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 11/3/22, 2:50 AM
Description: mouseClicked event handler

java.lang.NullPointerException: Cannot invoke "net.creeperhost.minetogether.polylib.gui.DropdownButton.m_6375_(double, double, int)" because "this.dropdownButton" is null
	at net.minecraft.client.gui.screens.ChatScreen.handler$bab000$onMouseClicked(ChatScreen.java:655) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:minetogether-common.mixins.json:chat.ChatScreenMixin,pl:mixin:A}
	at net.minecraft.client.gui.screens.ChatScreen.m_6375_(ChatScreen.java) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:minetogether-common.mixins.json:chat.ChatScreenMixin,pl:mixin:A}
	at net.minecraft.client.MouseHandler.m_168084_(MouseHandler.java:88) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries.mixins.json:MouseHandlerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:528) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:computing_frames,pl:epic_rei_lmao:B,pl:accesstransformer:B,xf:fml:configured:set_background_texture_2,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,xf:fml:configured:set_background_texture_2,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,xf:fml:configured:set_background_texture_2,pl:mixin:APP:kubejs-common.mixins.json:ScreenMixin,pl:mixin:APP:balm.mixins.json:ScreenAccessor,pl:mixin:APP:controlling.mixins.json:AccessScreen,pl:mixin:APP:patchouli_xplat.mixins.json:client.AccessorScreen,pl:mixin:APP:minetogether-common.mixins.json:chat.ScreenMixin,pl:mixin:APP:ae2.mixins.json:WrappedGenericStackTooltipModIdMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.MouseHandler.m_91530_(MouseHandler.java:85) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries.mixins.json:MouseHandlerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.MouseHandler.m_168091_(MouseHandler.java:185) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries.mixins.json:MouseHandlerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:103) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:computing_frames,pl:epic_rei_lmao:B,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B}
	at net.minecraft.client.MouseHandler.m_91565_(MouseHandler.java:184) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:MouseHandlerAccessor,pl:mixin:APP:supplementaries.mixins.json:MouseHandlerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) ~[lwjgl-glfw-3.2.2.jar%2348!/:build 10] {}
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar%2344!/:build 10] {}
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101) ~[lwjgl-glfw-3.2.2.jar%2348!/:build 10] {}
	at com.mojang.blaze3d.systems.RenderSystem.m_69495_(RenderSystem.java:204) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,re:classloading,pl:epic_rei_lmao:B,pl:mixin:APP:flywheel.mixins.json:RenderTexturesMixin,pl:mixin:A}
	at com.mojang.blaze3d.platform.Window.m_85435_(Window.java:333) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:mixin:APP:kubejs-common.mixins.json:WindowMixin,pl:mixin:APP:mixins.launchertray.json:WindowMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1068) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:traylauncher:loadingHookCoremod,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,xf:fml:traylauncher:loadingHookCoremod,pl:mixin:APP:kubejs-common.mixins.json:MinecraftMixin,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:tklib.mixin.json:client.MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.AccessorMinecraft,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:bookshelf.common.mixins.json:client.AccessorMinecraft,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:mixin.dynamic_asset_generator.json:MinecraftMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:traylauncher:loadingHookCoremod,pl:runtimedistcleaner:A,re:classloading,pl:epic_rei_lmao:B,pl:accesstransformer:B,xf:fml:traylauncher:loadingHookCoremod,pl:mixin:APP:kubejs-common.mixins.json:MinecraftMixin,pl:mixin:APP:balm.mixins.json:MinecraftMixin,pl:mixin:APP:tklib.mixin.json:client.MinecraftMixin,pl:mixin:APP:botania_xplat.mixins.json:client.AccessorMinecraft,pl:mixin:APP:ae2.mixins.json:PickColorMixin,pl:mixin:APP:ars_nouveau.mixins.json:light.ClientMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:bookshelf.common.mixins.json:client.AccessorMinecraft,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:mixin.dynamic_asset_generator.json:MinecraftMixin,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:APP:ars_nouveau.mixins.json:camera.MinecraftMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.84.jar%2317!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}
```

### Version

minetogether-forge-1.18.2-6.0.7.jar

### Minecraft Version

1.18.2

### ModLoader Version

Forge 40.1.84

### Additional Information

This is currently happening in the FTB StoneBlock 3 modpack, on version 1.0.1

Thanks for checking it!

**Comment by Raidobw2** [comment]
Hmm this seems to also happen when opening the game to lan

See https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1582, there is also a log in there (with the same stacktrace)

**Comment by ThePaul-T** [comment]
Thanks regen, we will get on it, so much testing and still managed to miss these

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by covers1624** [comment]
This has been fixed in dev.

**Comment by Raidobw2** [comment]
Yep no worries. Thanks for the quick answer!

**Comment by covers1624** [comment]
This has been fixed in MineTogether 6.0.8. Pending review on CurseForge.
